integration/class-yoast-seo.php

```diff
@@ -16,7 +16,7 @@ class Yoast_Seo {
 	 * Initialize the class, registering WordPress hooks.
 	 */
 	public static function init() {
-		\add_filter( 'site_status_tests', array( self::class, 'add_site_health_tests' ) );
+		\add_filter( 'site_status_tests', array( self::class, 'add_site_health_tests' ), 11 );
```

Comment:
```suggestion
		\add_filter( 'site_status_tests', array( self::class, 'add_site_health_tests' ), 11 ); // After Health_Check::add_tests().
```
Thanks! Let's document that for future reference